packages/core/build/config.js

```diff
@@ -75,6 +75,14 @@ const copyConfig = base => [
             return transformContentUrlBase(content, path, base);
         },
     },
+    {
+        from: path.resolve(__dirname, '../src/templates/app/apple-app-site-association'),
```

Comment:
@suyash-binary I don't know if these two underline prefix before `dirname` are intentional or not. What's the reason of it? 🤔

Reply:
> @suyash-binary I don't know if these two underscores prefix before `dirname` are intentional or not. What's the reason of it? 🤔
> I mean why 2 underscores?

This PR is not supposed to be merged as far as I know. I created this as a reference. 
Regarding your question, [__dirname](https://nodejs.org/docs/latest/api/modules.html#modules_dirname) is a magical nodejs thing. And most importantly I copy pasted, this file is full of __dirname 😬

Reply:
@Mohammad-fs This is like a global variable in Node.js, But the funny thing is it's `NOT` :))
This variable points to the current module directory name.

For example, if you are running your `something.js` file inside `User/jamshid/my-awsome-project`, This variable will return `User/jamshid/my-awsome-project` for you so you can find out where your project lives at. You can get the same with `path.dirname()` too.

Also, there is the `__filename` variable too which does a similar thing but it will include running file name as well, So the result will be `User/jamshid/my-awsome-project/something.js`.

Reply:
@farzin-fs Thanks for the explanations 🙏